src/pcp.rs

```diff
@@ -240,7 +248,45 @@ where
     /// "leader" share from the others. This is useful, for example, when some of the gadget inputs
     /// are constants used for both proof generation and verification.
     ///
-    /// TODO(cjpatton) Add an example once we have implemented a data type that uses this method.
+    /// ```
+    /// use prio::pcp::types::MeanVarUnsignedVector as MvuVec;
+    /// use prio::pcp::{decide, prove, query, Value, Proof, Verifier};
+    /// use prio::field::{split, FieldElement, Field64};
+    ///
+    /// use std::convert::TryFrom;
+    ///
+    /// let measurement = [1, 2, 3];
+    /// let bits = 8;
+    ///
+    /// let joint_rand = [Field64::rand(), Field64::rand(), Field64::rand()];
+    /// let query_rand = [Field64::rand()];
+    ///
+    /// let x: MvuVec<Field64> = MvuVec::new(bits, &measurement).unwrap();
+    /// let x_shares: Vec<MvuVec<Field64>> = split(x.as_slice(), 2)
+    ///     .into_iter()
+    ///     .enumerate()
+    ///     .map(|(i, data)| {
+    ///         let mut share = MvuVec::try_from((x.param(), data)).unwrap();
+    ///         share.set_leader(i == 0);
+    ///         share
+    ///     })
+    ///     .collect();
+    ///
+    /// let pf = prove(&x, &joint_rand).unwrap();
+    /// let pf_shares: Vec<Proof<Field64>> = split(pf.as_slice(), 2)
+    ///     .into_iter()
+    ///     .map(|data| Proof::from(data))
```

Comment:
nit: I'm not sure but I think this works, because `Proof::from` itself is already a function that takes `Vec<Field64>` and returns `Proof<Field64>`
```suggestion
    ///     .map(Proof::from)
```

Reply:
Oooh nice! Fixed here and elsewhere.